app/src/main/java/edu/uc/haflga/financialpal/AddTransaction.kt

```diff
@@ -4,7 +4,7 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import edu.uc.haflga.financialpal.ui.addtransaction.AddTransactionFragment
 
-class add_transaction : AppCompatActivity() {
+class AddTransaction : AppCompatActivity() {
```

Comment:
This suggestion aligns with proper naming conventions: UpperCamelCase for classes.